src/clarity/dropdown/dropdown.spec.ts

```diff
@@ -112,10 +120,16 @@ describe("Dropdown", () => {
 
     it("closes the menu when clicked outside of the host", () => {
         let dropdownToggle: HTMLElement = compiled.querySelector(".dropdown-toggle");
+        let outsideButton: HTMLElement = compiled.querySelector(".outside-click-test");
 
+        //check if the dropdown is closed
         expect(compiled.querySelector(".open")).toBeNull();
+
         //click outside the dropdown
-        fixture.nativeElement.click();
+        outsideButton.click();
+        //check if the click handler is triggered
+        expect(fixture.componentInstance.testCnt).toEqual(1);
+
         fixture.detectChanges();
```

Comment:
Just to be safe, detect changes after a user action and before the assertion. So `outsideButton.click()`, then `fixture.detectChanges()`, and after that your two `expect`.